src/literals.py

```diff
@@ -25,6 +25,7 @@
 
 ADMIN_USER = "operator"
 INTERNAL_USERS = [ADMIN_USER]
+SALT = "placeholder"
```

Comment:
**question:** Does this change? Feels unsafe.

Reply:
It doesn't really matter imo. We are already generating big random passwords to start with, so most of the benefits of a salted password are not that relevant in our case. Also, it makes easier to handle comparisons between existing authfile and expected one, since we don't need to remove hash and salt fields. Comparisons can be done as is.